base/Dockerfile

```diff
@@ -40,4 +40,9 @@ RUN chmod 755 /usr/local/bin/go-wrapper
 RUN ln -s /go/src/app /app
 WORKDIR /go/src/app
 
+ENV CLOUD_DEBUG_ENABLE "true"
```

Comment:
To me, most of the (future) value added by this image over the stock Go image is that the Stackdriver agents are preconfigured.